### PR TITLE
{phys}[gompi/2023a] Rivet v3.1.9, HepMC3 v3.2.6, fastjet v3.4.2, fastjet-contrib v1.053, siscone v3.0.6

### PR DESCRIPTION
**Pull request by APN-Pucky**
This PR is for https://github.com/EESSI/software-layer/pull/418
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/Rivet/Rivet-3.1.9-gompi-2023a-HepMC3-3.2.6.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+homepage = 'https://gitlab.com/hepcedar/rivet'
+description = """
+Rivet toolkit (Robust Independent Validation of Experiment and Theory)
+
+To use your own analysis you must append the path to `RIVET_ANALYSIS_PATH`.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2023a'}
+toolchainopts = {'optarch': True, 'pic': True}
```
Comment (boegel) [inline easybuild/easyconfigs/r/Rivet/Rivet-3.1.9-gompi-2023a-HepMC3-3.2.6.eb:20]: `optarch` is enabled by default, so this should become:

```suggestion
toolchainopts = {'pic': True}
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--sanity-check-only"

**Comment by boegel** [comment]
Going in, thanks @APN-Pucky!

**Comment by APN-Pucky** [comment]
Thanks to you @boegel . I am very excited about using it in my institute with eessi, since we can upgrade OSes w/o worrying about breaking the software.

**Pull request merged**